### ISSUE TITLE
The xscreensaver_bouncing_text doesn't work

### ISSUE THREAD

**Issue by JuliusLongmind** [Salamek/qiosk#1]
Hi Salamek!

After I successfully upgraded to the newest version the screensaver is stopped working with the following error:
https://pastebin.com/E9YTXJH0

Even when I start from shell as chromium-kiosk user I git this error, but when I exported the DISPLAY=:0, then it worked,

I hope this helps & and thank you for this nice piece of software,
Julius

**Comment by Salamek** [comment]
Hi, how are you starting xscreensaver_bouncing_text ? did you read https://github.com/Salamek/chromium-kiosk/tree/master/.docs#screensaver ?

**Comment by JuliusLongmind** [comment]
Yes, and if I remove the last 2 line from .screensaver file in /home/chromium-kiosk directory, then the function works. So I decided to login via ssh and run it from command prompt and while I was able to start other screensavers didn't start.
Also I used the --log --versbose for starting xscreensaver I got the same error.

**Comment by Salamek** [comment]
Hmm it works fine for me... 
How are you running it?

if just raw in shell it needs export DISPLAY=:0, but xscreensaver should handle that for you when running normally...
